TelCo.ColorCoder/getColor.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{  
+
+	class getColor
+	{
+		public static Color[] colorMapMajor;
+
+        public static Color[] colorMapMinor;
+
+        static getColor()
+        {
+            colorMapMajor = new Color[] { Color.White, Color.Red, Color.Black, Color.Yellow, Color.Violet };
+            colorMapMinor = new Color[] { Color.Blue, Color.Orange, Color.Green, Color.Brown, Color.SlateGray };
+        }
+
+        public static Pair GetColorFromPairNumber(int pairNumber)

```

Comment:
Why can't `GetColorFromPairNumber` also be in the `Pair` class?
Rename the `Pair` class to be more specific to its contents.

Reply:
i have made all changes mentioned, please check if i have met the expectations!